client/src/templates/Introduction/components/CertChallenge.js

```diff
@@ -118,27 +146,35 @@ const CertChallenge = ({
       ? verifyCert(certSlug)
       : createFlashMessage(honestyInfoMessage);
   };
-
   return (
     <div className='block'>
-      {(!isCertified || !canViewCert) && (
-        <CertificationCard
-          i18nCertText={i18nCertText}
-          isProjectsCompleted={isProjectsCompleted}
-          steps={steps}
-          stepState={stepState}
-          superBlock={superBlock}
-        />
-      )}
-      <Button
-        block={true}
-        bsStyle='primary'
-        disabled={!canClaim.status || (isCertified && !canViewCert)}
-        href={certLocation}
-        onClick={createClickHandler(certSlug)}
-      >
-        {isCertified ? t('buttons.show-cert') : t('buttons.claim-cert')}
-      </Button>
+      {userLoaded &&
+        isSignedIn &&
+        (!isCertified || (!canViewCert && userChecked)) && (
```

Comment:
```suggestion
      {userLoaded &&
        isSignedIn &&
        (!isCertified || (!canViewCert && userChecked)) && (
```
This is a bit complicated.  I'd recommend moving this into the render body and giving it a name.  `showCertificationCard` is probably fine.